src/libs/sfdk/buildengine.cpp

```diff
@@ -439,7 +439,9 @@ bool BuildEnginePrivate::initVirtualMachine(const QUrl &vmUri)
     Q_Q(BuildEngine);
     Q_ASSERT(!virtualMachine);
 
-    virtualMachine = VirtualMachineFactory::create(vmUri);
+    VirtualMachine::Features unsupportedFeatures = VirtualMachine::Snapshots;
+
+    virtualMachine = VirtualMachineFactory::create(vmUri, ~unsupportedFeatures);
```

Comment:
For some reason the old logic was easier to understand (for me). But I *think* this works.